go.mod

```
@@ -5,6 +5,7 @@ go 1.20
 require (
 	github.com/Depado/ginprom v1.7.4
 	github.com/Masterminds/semver/v3 v3.2.0
+	github.com/Masterminds/sprig v2.22.0+incompatible
```

Is there a newer version of this module that uses `github.com/Masterminds/semver/v3` which we already import?

---

Getting this if I try to update to latest version
```go: errors parsing go.mod:
/Users/gheorghestrimtu/Documents/chainlink/git/chainlink/go.mod:9:2: require github.com/Masterminds/sprig: version "v3.2.3" invalid: should be v0 or v1, not v3
```

---

I think you have to include the `/v3` suffix too. I guess they didn't follow that pattern with v2?
```suggestion
	github.com/Masterminds/sprig/v3 v3.2.3
```

---

worked